src/plugins/usage_collection/public/components/track_application_view/track_application_view_component.tsx

```diff
@@ -15,26 +15,34 @@ interface Props extends TrackApplicationViewProps {
   applicationUsageTracker?: IApplicationUsageTracker;
 }
 
-export class TrackApplicationViewComponent extends Component<Props> {
-  onClick = () => {
+export class TrackApplicationViewComponent extends React.Component<Props> {
+  private parentNode: (Node & ParentNode) | null | undefined;
+
+  onClick = (e: MouseEvent) => {
     const { applicationUsageTracker, viewId } = this.props;
-    applicationUsageTracker?.updateViewClickCounter(viewId);
+    this.parentNode = this.parentNode || ReactDOM.findDOMNode(this)?.parentNode;
+    if (this.parentNode === e.target || this.parentNode?.contains(e.target as Node | null)) {
+      applicationUsageTracker?.updateViewClickCounter(viewId);
+    }
   };
 
   componentDidMount() {
     const { applicationUsageTracker, viewId } = this.props;
     if (applicationUsageTracker) {
       applicationUsageTracker.trackApplicationViewUsage(viewId);
-      ReactDOM.findDOMNode(this)?.parentNode?.addEventListener('click', this.onClick);
+      setTimeout(() => {
+        this.parentNode = ReactDOM.findDOMNode(this)?.parentNode;
```

Comment:
This is where the console error is being thrown from.
In chrome console sources, with debug breakpoints, I get the following on the second pass through `componentDidUpdate`:
![Screen Shot 2021-07-22 at 12 21 01](https://user-images.githubusercontent.com/11909450/126697482-aa6d9f6e-23fc-416b-9c4a-3e92f4b403f3.png)

Reply:
fixed